RoddyCore/src/de/dkfz/roddy/config/validation/DefaultValidator.groovy

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2016 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.roddy.config.validation
+
+import de.dkfz.roddy.config.Configuration
+import de.dkfz.roddy.config.ConfigurationIssue
+import de.dkfz.roddy.config.ConfigurationValue
+import de.dkfz.roddy.config.EnumerationValue
+import groovy.transform.CompileStatic
+
+import static de.dkfz.roddy.config.ConfigurationIssue.ConfigurationIssueTemplate.*
+
+/**
+ * This validator check values of type
+ * string (not really checked)
+ * integer
+ * boolean
+ */
+@CompileStatic
+class DefaultValidator extends ConfigurationValueValidator {
+    private static final de.dkfz.roddy.tools.LoggerWrapper logger = de.dkfz.roddy.tools.LoggerWrapper.getLogger(DefaultValidator.class.getSimpleName())
+
+    boolean result = false
+
+    DefaultValidator(Configuration cfg) {
+        super(cfg)
+    }
+
+    @Override
+    boolean validate(ConfigurationValue configurationValue) {
+        super.errors.clear()
+        super.warnings.clear()
+        EnumerationValue ev = configurationValue.getEnumerationValueType()
+
+        boolean result = checkDataTypes(configurationValue, ev)

```

Comment:
As type this is an EnumerationValue. In semantics it is a configurationValueType. The variable name does not express this.
```suggestion
        EnumerationValue configurationValueType = configurationValue.getEnumerationValueType()
        boolean result = checkDataTypes(configurationValue, configurationValueType)
```

Reply:
Changed it.